api-compact.yaml

```diff
@@ -27,7 +30,7 @@ apigateway_responses: &apigateway_responses
   500:
     statusCode: "500"
 
-apigateway_responses_with_mapping_template: &apigateway_responses_with_mapping_template
+apigateway_responses_with_mapping_template_for_id: &apigateway_responses_with_mapping_template_for_id
```

Comment:
@saleem-latif Makes sense -- but the thing is this could end up confusing someone who sees that it's not being used for the `/enterprise/api/enterprise_catalogs/{uuid}` endpoint, because that one doesn't use the `id` format.